modules/shared/components/molecules/postFooter/PostFooter.module.css

```diff
@@ -0,0 +1,4 @@
+.container {
+  @apply flex justify-between items-center
```

Comment:
just tiny little change.
the children elements which is (votes and share) are not centered on the cross axis. so just remove "items-center" and it will return to default which is flex-start.